scripts/shell.py

```diff
@@ -0,0 +1,5 @@
+from app import create_app
+
+create_app()
+
+from app.models import *
```

Comment:
Unfortunately this won't work. Lets just skip this for now and think about it more later :)